lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -50,7 +50,6 @@
   // Cover all columns and the third through last rows of the nx-page grid template
   grid-area: 3 / 1 / -1 / -1;
   max-width: var(--nx-width-page-max);
-  overflow-y: hidden;
```

Comment:
I'm not entirely sure this is the right solution.
I tried looking for things that might break removing this, but couldn't find anything.
Not entirely sure the purpose behind this, let me know your thoughts!

With this removed the dropdown extends the scrollbar down when opened:

<img width="408" alt="Screen Shot 2022-10-11 at 3 26 23 PM" src="https://user-images.githubusercontent.com/10160030/195181113-473a252f-8c97-4025-a7a6-f959ad9ae698.png">

Reply:
> I'm not entirely sure this is the right solution.
I tried looking for things that might break removing this, but couldn't find anything.
Not entirely sure the purpose behind this, let me know your thoughts!

Do section-scrolling layouts still work?

My assumption had been that we would override this back to visible in `.nx-html--page-scrolling .nx-page-content`, but I too am skeptical that it's safe to remove in when section-scrolling.

Reply:
Hmm I remember it did work before and testing it again and it looks like section-scrolling layouts still works 🤔.
I've updated the implementation a bit based on your comment here: cf28564974bf32e1d942a8082ad1dffa987e6148

Going to look into it a bit more!